feature-libs/cart/added-to-cart-toast/core/models/added-to-cart-toast.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Images } from 'projects/core/src/model';
+
+export interface CartToastItem {
+  productName: string;
+  quantity: number;
+  unitPrice: string;
+  baseClass: string;
+  images?: Images;
+  timeoutRef?: ReturnType<typeof setTimeout>;
```

Comment:
What is the purpose of this?

Reply:
This is a reference to the timeout of the toast. In the case that a user clicks on 'Continue Shopping' or 'Go to Cart', this reference will be used for the cancel the timeout.